x-pack/packages/security-solution/features/src/cases/v2_features/kibana_sub_features.ts

```diff
@@ -121,8 +121,8 @@ export const getCasesSubFeaturesMapV2 = ({
             ),
             includeIn: 'all',
             savedObject: {
-              all: [],
-              read: [],
+              all: [...savedObjects.files],
```

Comment:
@cnasikas - I thought we _did_ want `createComment` and attachments to still remain interconnected. I Can revert these changes if we'd rather not.